datapath.go

```diff
@@ -35,32 +36,49 @@ func (dp *Datapath) sendLoop() {
 		msg := <-(dp.sendBuffer)
 		// serialize data
 		byteData := (*msg).Serialize()
-		_, err := dp.conn.Write(byteData)
-		if err != nil {
-			fmt.Println("failed to write conn")
-			fmt.Println(err)
-			return
+		writePos := 0
+		for writePos != len(byteData) {
+			written, err := dp.conn.Write(byteData[writePos:])
```

Comment:
Took a closer look at dp.conn.Write with @dmakarov-hystax. Seems that it already has a such implementation inside. 
This changes should be removed as unnecessary.

Reply:
Fixed.